src/effects/effectchain.cpp

```diff
@@ -95,9 +95,6 @@ EffectChainPointer EffectChain::clone(EffectChainPointer pChain) {
     pClone->setEnabled(pChain->enabled());
     pClone->setName(pChain->name());
     pClone->setMix(pChain->mix());
-    foreach (const ChannelHandleAndGroup& handle_group, pChain->enabledChannels()) {
-        pClone->enableForChannel(handle_group);
-    }
```

Comment:
This was interfering with keeping channels enabled when switching between custom effects chains made with Deere and the premade chains in LateNight. EffectsManager::processEffectsResponses was printing an error indicating the EffectRequest to the engine failed. This code is redundant with calls to EffectChain::enable/disableForChannel in EffectChainSlot::loadEffectChain, which gets called after EffectChain::clone in EffectRack::loadNext/PrevChain.